frontend/src/ModelOverview.tsx

```diff
@@ -69,21 +71,51 @@ function ModelOverview({ version }: ModelOverviewProps) {
             </Box>
           </Box>
           <Box sx={{ p: 2 }}>
-            <Typography variant='h6'>{schema?.schema.properties.contacts.properties.uploader.title}</Typography>
+            <Stack spacing={0.5} direction='row'>
+              <Typography variant='h6'>{schema?.schema.properties.contacts.properties.uploader.title}</Typography>
 
+              <HelpPopover
```

Comment:
This can be simplified if you set default values within your HelpPopover component (as suggested here: https://github.com/gchq/Bailo/pull/512/files#r1206667274)
```
              <HelpPopover>
                <Typography sx={{ p: 1 }}>
                  {schema?.schema.properties.contacts.properties.uploader.description}
                </Typography>
              </HelpPopover>
```